### DISCUSSION TITLE
Mumble/murmur support

### DISCUSSION THREAD

**Discussion by pataquets** [baresip/baresip#2200] [General]
By looking at the module list, looks like it's not supported. Is it?
It would be useful to create a SIP-to-Mumble gateway, for example.
If the feature is within scope for the project, I can edit the issue to convert it to a feature request (although I can't code it myself 😞)

**Comment by alfredh** [comment]
Please use "Discussions" for questions:

https://github.com/baresip/baresip/wiki/Guidelines-for-reporting-Issues

**Comment by pataquets** [comment]
I wasn't aware of the guidelines. My apologies. Also, I've noticed the guidelines don't mention to Github discussions for Q&A, but a mailing list on Google Groups. I was ready to edit the wiki, but I wonder if it is the ML deprecated in favour of Gh discussions? are both OK? I'll be happy to fix it.
